how-to/interact-database.rst

```diff
@@ -38,7 +38,7 @@ You can also make the connection manually from within the ``web`` container, for
 
     docker-compose run --rm web psql -h postgres -U postgres db
 
-As well as ``psql`` you can run commands such as ``pg_dump`` and ``pg_restore``. This is useful
+The same as ``psql``, you can also run commands such as ``pg_dump`` and ``pg_restore``. This is useful
 for a number of :ref:`common operations <common-db-operations>`, below.
 
```

Comment:
Was better previously, please revert this change

Reply:
Done